src/main/java/spark/Service.java

```diff
@@ -162,6 +162,26 @@ public synchronized Service secure(String keystoreFile,
         sslStores = SslStores.create(keystoreFile, keystorePassword, truststoreFile, truststorePassword);
         return this;
     }
+ 
+    public synchronized Service secure(String keystoreFile,
+							           String keystorePassword,
+							           String truststoreFile,
+							           String truststorePassword,
+							           String[] excludeProtocols,
```

Comment:
Traditionally, when configuring nginx and Apache web servers, they work with include lists because it's treated as a whitelist.  Using a blacklist/exclude list has the downside that new protocols and ciphers automatically become enabled.

Checkout: http://download.eclipse.org/jetty/stable-9/apidocs/org/eclipse/jetty/util/ssl/SslContextFactory.html#setIncludeCipherSuites-java.lang.String...-
void setIncludeCipherSuites(String... cipherSuites)
void	setIncludeProtocols(String... protocols)

Using an include list also makes it much easier for an Ops team to work with SparkJava.